src/marqo/core/unstructured_vespa_index/unstructured_document.py

```diff
@@ -103,6 +110,9 @@ def from_marqo_document(cls, document: Dict, filter_string_max_length: int) -> "
         instance.fields.vector_counts = len(instance.fields.vespa_embeddings)
         return instance
 
+    def _is_large_int(value, low_threshold=2**31, high_threshold=2**64):
```

Comment:
return type, `bool`. It would be good to add doc strings explaining why we need to do this check. Really good practice to start the method with `_`